src/pages/home-page-slides/slides-order/slides-order.js

```diff
@@ -0,0 +1,186 @@
+    dragItemNode.current.removeEventListener('dragend', handleDragEnd);
+    dragItemNode.current = null;
+  };
+
+  const getStyles = (item) => {
+    if (dragItem.current.grpI === item.grpI && dragItem.current.itemI === item.itemI) {
+      return `${styles.dndItem} ${styles.current}`;
+    }
+    return styles.dndItem;
+  };
+  const changeHandler = () => setDraggable(true);
+  const saveHandler = ()=>{
+    const availableArray = []
+    const nonAvailableArray = []
+    list.forEach(el=>{
+
+      if(el.title==='available'){
+        el.items.map((availableSlide,index) => {
+          const {order,show,...rest} = availableSlide
+          availableArray.push({id:availableSlide._id,slide:{order:+index+1, show:true, ...rest}})
+          return {order,show}
+        });
+      }
+      if(el.title==='nonAvailable'){
+        el.items.map((nonAvailableSlide) => {
+          const {order,show,...rest} = nonAvailableSlide
+          nonAvailableArray.push({id:nonAvailableSlide._id,slide:{order:0, show:false, ...rest}})
+          return {order,show}
+        });
+      }
+    });
+    const newSlideItems = availableArray.concat(nonAvailableArray)
+    newSlideItems.forEach(item=>dispatch(updateSlidesOrder({id:item.id, slide:{order:item.slide.order, show:item.slide.show}})))
+    const arrayToStore = []
+    newSlideItems.forEach(el=>arrayToStore.push(el.slide))
+    dispatch(setSlides(arrayToStore))
+    setDraggable(false)
+  }
+  const dnbContainer = drugAndDropList.length
+    ? list.map((grp, grpI) => (
+      <Card key={grp.title}
+        elevation={2}
+        onDragEnter={dragging && !grp.items.length ? (e) => handleDragEnter(e, { grpI, itemI: 0 }) : null}
+        className={styles.dndGroup}
+      >
+        <Typography variant='h1' className={styles.slideTitle}>
+          {
+            grp.title ==='available'
+              ? slidesTranslations.available
+              : slidesTranslations.nonAvailable
+          }
+        </Typography>
+        {grp.items.map((item, itemI) => (
+          <Paper
+            draggable={draggable}
+            elevation={5}
+            onDragStart={(e) => handlerDragStart(e, { grpI, itemI })}
+            onDragEnter={dragging ? (e) => {
+              handleDragEnter(e, { grpI, itemI });
+            } : null}
+            className={dragging ? getStyles({ grpI, itemI }) : styles.dndItem}
+            key={item._id}
+          >
+            <Avatar variant='square'
+              className={styles.square}
+              src={
+                `${IMG_URL}${item.images.small}`
+              }
+              color='primary'
+            >
+              <ImageIcon />
+            </Avatar>
+            <div className={styles.slideContent}>
+              <div>
+                <h3>{item.title[0].value}</h3>
+                <p>{item.description[0].value}</p>
+              </div>
+              <p className={styles.discoverMore}> {discoverMoreTitle}
+                <span>&#8594;</span></p>
```

Comment:
Try to replace this hardcode unicode with a materialUI icon or add it to the config file

Reply:
Fixed